packages/Jms/src/Rector/Property/JmsInjectAnnotationRector.php

```diff
@@ -151,13 +154,13 @@ private function resolveType(Node $node): ?string
     private function resolveServiceName(PhpDocTagNode $phpDocTagNode, Node $node): ?string
     {
         $injectTagContent = (string) $phpDocTagNode->value;
-        $match = Strings::match($injectTagContent, '#(\'|")(?<serviceName>[\w\._-]+)(\'|")#');
+        $match = Strings::match($injectTagContent, '#(\'|")(?<serviceName>[^\'|"]+)(\'|")#');
```

Comment:
This can be replaced by `.*?` - "as much as possible  up-to following match"

```diff
-(\'|")(?<serviceName>[^\'|"]+)(\'|")
+(\'|")(?<serviceName>.*?)(\'|")
```

Reply:
sure